app/src/test/java/com/google/droidjump/ExampleUnitTest.java

```diff
@@ -30,4 +30,4 @@
     public void addition_isCorrect() throws Exception {
         assertEquals(4, 2 + 2);
```

Comment:
why is this file here?

Reply:
This is an example unit test that was created along with empty activity. Let it be there until we have real unit tests.